pkg/util/flag/flags.go

```diff
@@ -25,6 +25,7 @@ import (
 	"k8s.io/klog"
 
 	utilnet "k8s.io/apimachinery/pkg/util/net"
+	netutils "k8s.io/utils/net"
```

Comment:
@vegemitecheesetoast I think that you are missing this import in other files.

Reply:
Ah, I thought this was automatic. Will put in files!

Reply:
I think I fixed it.